core/DataTable/Row.php

```diff
@@ -521,7 +521,13 @@ private function getColumnValuesMerged($operation, $thisColumnValue, $columnToSu
                 $newValue = null;
                 break;
             case 'max':
-                $newValue = max($thisColumnValue, $columnToSumValue);
+                if (!$thisColumnValue) {
+                    $newValue = $columnToSumValue;
+                } elseif (!$columnToSumValue) {
+                    $newValue = $thisColumnValue;
+                } else {
+                    $newValue = max($thisColumnValue, $columnToSumValue);
+                }
```

Comment:
This was added to handle min and max values the same. Without that all max values would always be `0`, even if there was not a single value tracked.

Reply:
Is the actual logic behind the code as follows?

```php
            case 'max':
                if (!is_numeric($thisColumnValue) && !is_numeric($columnToSumValue)) {
                    $newValue = null;
                } else {
                    $newValue = max(($thisColumnValue ?? 0), ($columnToSumValue ?? 0));
                }
```

Reply:
Thanks for pointing to that. I actually didn't think too much about that. I will try to improve the code